### DISCUSSION TITLE
How can I get the state of text written inside the select search bar?

### DISCUSSION THREAD

**Discussion by sifu442** [filamentphp/filament#14651] [Help]
### Package

Form builder

### Package Version

3.2

### How can we help you?

In the picture, Srart typing to search. I want the state of the written text inside of the search bar and send it to another field. How can I achieve that?
![grouped](https://github.com/user-attachments/assets/537aa505-1110-46da-a13d-749bdfa006ce)

**Comment by leandrocfe** [comment]
I don't think so, but could you explain why are you trying to do this?
